Fix for french translation => Update package.nls.fr.json

cobra91 opened the pull request:
## Context

Fix a translation for french users

cobra91 committed:
Update package.nls.fr.json

hassoncs reviewed (commented):
Thank you for opening this PR, @cobra91! I think the length limit for `displayName` is 50 characters. Maybe we could find a shorter translation? Our old `displayName` is _right_ up against the 50 character limit 😬

```
"Kilo Code AI Agent (Cline / Roo fonctionnalités combinées)".length
58
"Kilo Code AI Agent (Cline / Roo features combined)".length
50
```

cobra91 committed:
Merge branch 'main' into patch-1

cobra91 committed:
Update package.nls.fr.json

lesser than 50 character

cobra91 committed:
Update common.json

use of informal address instead of formal address
In French, "billion" = a thousand billion (10¹²), not a billion like in English!

cobra91 committed:
Update kilocode.json

use of informal address instead of formal address

chrarnoldus committed:
Create ninety-lemons-rest.md

chrarnoldus committed:
Merge branch 'main' into patch-1

cobra91 commented on src/i18n/locales/fr/common.json:
```diff
@@ -145,7 +145,7 @@
 	"input": {
 		"task_prompt": "Que doit faire Kilo Code?",
-		"task_placeholder": "Écris ta tâche ici"
+		"task_placeholder": "Écrivez votre tâche ici"
```
Hum first time I see use of the informal form of address instead of the formal form of address
